example/lib/main.dart

```diff
@@ -43,6 +46,11 @@ class _MyHomePageState extends State<MyHomePage> {
     if (input == '\r') {
       terminal.write('\r\n');
       terminal.write('\$ ');
+    } else if (input.codeUnitAt(0) == 127) {
+      terminal.buffer.eraseCharacters(1);
+      terminal.buffer.backspace();
+      terminal.refresh();
+      return;
```

Comment:
nit: you don't need return here, although if you end up adding more logic below line 57 I guess you would. Otherwise this seems fine to me.

It might also be worth adding a comment that this is needed because this example is not connected to a backend that handles it, but if you connected to a "real" backend it would be handling this for you.